lib/ProxySQL_Config.cpp

```
@@ -2320,6 +2361,63 @@ int ProxySQL_Config::Read_PgSQL_Servers_from_configfile(std::string& error) {
+			if (line.lookupValue("min_lag_ms", min_lag_ms)==false) min_lag_ms=30;
+			if (line.lookupValue("lag_num_checks", lag_num_checks)==false) lag_num_checks=1;
+			line.lookupValue("active", active);
+			line.lookupValue("comment", comment);
+			line.lookupValue("domain_name", domain_name);
+			char *o1=strdup(comment.c_str());
+			char *o=escape_string_single_quotes(o1, false);
+			char *p1=strdup(domain_name.c_str());
+			char *p=escape_string_single_quotes(p1, false);
+			const size_t query_len = strlen(q)+strlen(o)+strlen(p)+256;
+			char *query=(char *)malloc(query_len);
+			snprintf(query, query_len, q, writer_hostgroup, reader_hostgroup, active, aurora_port, p, max_lag_ms, check_interval_ms, check_timeout_ms, writer_is_also_reader, new_reader_weight, add_lag_ms, min_lag_ms, lag_num_checks, o);
```

Cppcheck (2.21.0)

[warning] If memory allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfMemory)